Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -623,7 +654,18 @@ int IORead(HANDLE& dev_handle, OVERLAPPED& hid_overlap_read, u8* buf, int index)
     }
   }
 
-  return bytes + 1;
+  // ReadFile will always return 22 bytes read.
+  // So we need to calculate the acutal report size by its report id
```

Comment:
There are some minor typos here; it should be "_actual_ report size by its report _ID_"